cli/cli.py

```diff
@@ -71,12 +71,43 @@ def upload_assay(assay, xlsx):
     """
     Upload data for an assay.
     """
-    upload.upload_assay(assay, xlsx)
+    upload.run_upload(assay, xlsx)
+
+
+#### $ cidc analyses ####
+@click.group()
+@click.option("-i", "--ignore", default=None, hidden=True)
+def analyses(ignore):
```

Comment:
maybe this should be an option of the whole cli thing, not per assay or analysis?